Example/Controllers/Extensions/ViewController+MiniAppMessageProtocol.swift

```diff
@@ -54,4 +54,14 @@ extension ViewController: MiniAppMessageDelegate, CLLocationManagerDelegate {
         break
         }
     }
+
+    func getHostEnvironmentInfo(completionHandler: @escaping (Result<MAHostEnvironmentInfo, MASDKError>) -> Void) {
+        completionHandler(
+            .success(MAHostEnvironmentInfo(
+                platformVersion: "5.0.0",
+                hostVersion: "2.7.0",
+                sdkVersion: "3.6.0"
```

Comment:
I think there's something smarter to do with your `MAHostEnvironmentInfo` object. Host version and SDK version are already retrieved in `Environment.appVersion` and `Environment.sdkVersion`, there may be no need to provide them statically

Reply:
@Climbatize good point, I replaced the default implementation so it's using appVersion & sdkVersion but can be overridden by the host if necessary